crypto/nrf_oberon/include/ocrypto_aes_key.h

```diff
@@ -23,9 +23,9 @@
 extern "C" {
 #endif
 
-#define ocrypto_aes128_KEY_BYTES ((size_t) 16)  //!< AES-128 key size in bytes.
-#define ocrypto_aes192_KEY_BYTES ((size_t) 24)  //!< AES-192 key size in bytes.
-#define ocrypto_aes256_KEY_BYTES ((size_t) 32)  //!< AES-256 key size in bytes.
+#define ocrypto_aes128_KEY_BYTES ((size_t) 16)  //!< AES-128 key size in bytes
```

Comment:
FYI the punctuation-at-end might lack consistency (I don't mind either way).

Reply:
The //!< is for headers, they correspond to function/macro briefs which will be lead to their labels being a top-level grouping, hence they should not have punctuation.